assets/js/base/components/product-grid-item/product-image/index.js

```diff
@@ -6,43 +6,49 @@ import PropTypes from 'prop-types';
 /**
  * Internal dependencies
  */
-import {
-	PLACEHOLDER_IMG_SRC,
-	THUMBNAIL_SIZE,
-} from '@woocommerce/block-settings';
+import { PLACEHOLDER_IMG_SRC } from '@woocommerce/block-settings';
 
-const ProductImage = ( { className, product = {} } ) => {
-	let image = null;
+const ProductImage = ( { className, images } ) => {
+	if ( images && images.length ) {
+		const mainImage = images[ 0 ];
+		const aspectRatio = ( mainImage.height / mainImage.width ) * 100;
```

Comment:
Is width guaranteed to be > 0?

Reply:
Removed in b354d469710c58d0f9fcc88c69a1f02a2380d2f8.